indico/core/signals/rb.py

```diff
@@ -25,3 +25,22 @@
 rooms_fetched = _signals.signal('rooms_fetched', """
```

Comment:
can you remove this signal and the place where we're calling it since it's not needed anymore?

Reply:
nevermind, removed in master by #3841